### PR TITLE
va-link: Replacing external icon with text

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://3303-external-link-update--65a6e2ed2314f7b8f98609d8.chromatic.com

---
## Description
Closes [#3303](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3303)

## QA Checklist
- [X] Component maintains 1:1 parity with design mocks
- [X] Text is consistent with what's been provided in the mocks
- [X] Component behaves as expected across breakpoints
- [ ] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [X] Tab order and focus state work as expected
- [X] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [X] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [X] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Screenshots
Screenshot 2024-11-11 at 11 40 07 AM

## Acceptance criteria
- [ ] QA checklist has been completed
- [X] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/src/components/va-link/va-link.tsx [hunk 1]
```diff
@@ -285,8 +285,7 @@ export class VaLink {
             onClick={handleClick}
             target="_blank"
           >
-            {text}
-            <va-icon class="external-link-icon" icon="launch"></va-icon>
+            {text} (opens in a new tab)
```
[the remark this answers is not in the document]
Comment (Andrew565) [reply packages/web-components/src/components/va-link/va-link.tsx:288]: Can do!

**Review by jamigibbs — COMMENTED** [review]
Can you check what's up with `va-privacy-agreement`? It doesn't look like it's updated: https://3303-external-link-update--65a6e2ed2314f7b8f98609d8.chromatic.com/?path=/docs/uswds-va-privacy-agreement--docs


Screenshot 2024-11-12 at 9 16 29 AM

Also is it possible to keep the period from dropping to the next line in `va-statement-of-truth`?


Screenshot 2024-11-12 at 9 15 47 AM

**Comment by Andrew565** [comment]
@jamigibbs Looks like the privacy agreement wasn't using the va-link component, just a hard-coded link and va-icon. I'll update it, thanks for catching that!

I'll see what I can do about the statement of truth dropped period, that period exists outside of the va-link component so I'll have to mod the va-statement-of-truth component someway somehow.

**Review by rsmithadhoc — APPROVED** [review]

**Review by jamigibbs — APPROVED** [review]

File: packages/web-components/src/components/va-privacy-agreement/test/va-privacy-agreement.e2e.ts [hunk 2]
```diff
@@ -13,11 +13,7 @@ describe('va-privacy-agreement', () => {
          <va-checkbox class="hydrated" id="checkbox">
            <span class="description" slot="description">
              Please read and accept the
-             <a href="/privacy-policy/" target="_blank">
-               privacy policy
-               <va-icon class="hydrated privacy-policy-icon"></va-icon>
-               <span class="usa-sr-only">opens in a new window</span>
-             </a>.
+             <va-link class="hydrated"></va-link>.
```
[the remark this answers is not in the document]
Comment (Andrew565) [reply packages/web-components/src/components/va-privacy-agreement/test/va-privacy-agreement.e2e.ts:16]: Yeah, I was surprised by that too. Good thing we still have separate e2e tests for the va-link component itself to make sure those props work!

**Pull request merged**